2022/incoming/looking-at-the-christmas-numbers.pod

```diff
@@ -0,0 +1,151 @@
+            title        => 'gray20',
+        },
+    );
+    $g->png("bars1.png");
+
+=end perl
+
+=begin html
+<img src="https://raw.githubusercontent.com/lichtkind/Chart/main/dev/example/advent/bars1.png"    alt="first bar chart"  width="300" height="300">
+=end html
+
+
+=head2 Finding Colors
+
+The only issue now are the dataset colors, which are optimized for contrast (they scream).
+Intuitive would be to have shades of the same color to signify that neighbouring
+bars are about the same person, only some numbers are older than others.
+To achieve that pick some appealing color from the table in the lower half of 
+L< Graphics::Toolkit::Color::Constant>. I like C<'royalblue3'> of the I<CSS>
+standard. It also has the advantage of providing a workable gradient (list of
+shades) for free. So just add:
+
+=begin perl
+
+        colors => { ..
+            dataset0     => 'royalblue3',
+            dataset1     => 'royalblue2',
+            dataset2     => 'royalblue1',
+        },
+
+=end perl
+
+The only counterintuitive part of this code is to understand that the first
+C<add_dataset> only adds axis labels and the following C<add_dataset> is 
+C<dataset0>.
+
+=begin html
+<img src="https://raw.githubusercontent.com/lichtkind/Chart/main/dev/example/advent/bars2.png"    alt="first bar chart"  width="300" height="300">
+=end html
+
+=head2 Customize Colors
+
+Now the artist in me says: not bad, but the middle blue should be a little darker.
+Allright, let's load the whole toolkit (a direct dependency of C<Chart>) and import
+also the helper function C<color>, to keep it all short and sweet. Please make sure
+to have the latest version installed to use that feature.
+
+=begin perl
+
+    use Graphics::Toolkit::Color 'color';
+     ..
+
+        colors => { ..
+            dataset1     => color('royalblue3')->add( { Lightness => -4 } ),
+
+=end perl
+
+=begin html
+<img src="https://raw.githubusercontent.com/lichtkind/Chart/main/dev/example/advent/bars3.png"    alt="first bar chart"  width="300" height="300">
+=end html
+
+=head2 Open the Color Factory
+
+I'm good now, but what would a super picky designer, who just came from the 
+I<New York Fashion Week> say? Maybe something like: how lame - I want a gradient from
+I<Glacier Lake> to I<Harbor Blue>. Alright - let's check quickly if 
+L<Graphics::ColorNames::PantoneReport> is installed by C<grep> 'ing the
+result of C<Graphics::ColorNames::all_schemes> and if so, create our own
+three color gradient with the colors from this summers I<Pantone Report>.
+(Other standards like I<X>, I<HTML> or I<VACCC> are available too
+ - just look at L<Bundle::Graphics::ColorNames>).
+
+=begin perl
+
+    my @gradient = color('PantoneReport:Glacier_Lake')
+                         ->gradient_to('PantoneReport:Harbor_Blue', 3);
+     ..
+        colors => { ..
+            dataset0     => $gradient[0],
+            dataset1     => $gradient[1],
+            dataset2     => $gradient[2],
+
+=end perl
+
```

Comment:
Put a sentence here

Reply:
rewrote the ending , better explanation, better flowing text also added there sentence. even some one more little christmas hint.

Reply:
Thank you, you rock